torchaudio/prototype/ctc_decoder/ctc_decoder.py

```diff
@@ -257,3 +264,57 @@ def lexicon_decoder(
         sil_token=sil_token,
         unk_word=unk_word,
     )
+
+
+def download_pretrained_files(
+    path: Union[str, Path],
+    dataset: str,
+    lm: Optional[str] = None,
+) -> PretrainedFiles:
```

Comment:
The configuration of `dataset` + `lm` seems to be specific to the librispeech cases that we know, and I do not think this is future proof.

Imagine a hypothetical case where we want to add pertained model that is trained on some subset of a dataset. say `librispeech-dev100-3gram`. Do we want to add `subset` parameter? What about some other specifier that we do not know yet?

So I think it's more future proof to use single key for specifying the target model.
That way I think that the API will be more straightforward to users as well.